diff -urN zapping-0.7.0.orig/zapping_setup_fb/Makefile.am zapping-0.7.0/zapping_setup_fb/Makefile.am
--- zapping-0.7.0.orig/zapping_setup_fb/Makefile.am	Thu Mar 27 13:23:23 2003
+++ zapping-0.7.0/zapping_setup_fb/Makefile.am	Thu Mar 27 13:24:20 2003
@@ -13,8 +13,7 @@
 if SUID_ZSFB
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
 	$(mkinstalldirs) $(DESTDIR)$(sbindir)
-	$(INSTALL) -m4755 -o $(ZSFB_OWNER) -g $(ZSFB_GROUP) \
-	  zapping_setup_fb $(DESTDIR)$(sbindir)
+	$(INSTALL) zapping_setup_fb $(DESTDIR)$(sbindir)
 	ln -sf $(DESTDIR)$(sbindir)/zapping_setup_fb \
 	  $(DESTDIR)$(bindir)/zapping_setup_fb
 else
