--- zapping-0.7.1/configure.in.orig	2004-09-14 06:46:27.000000000 +0200
+++ zapping-0.7.1/configure.in	2004-09-19 12:51:19.633035072 +0200
@@ -626,7 +626,7 @@
   -DPACKAGE_DATA_DIR=\\\"\${datadir}\\\" \
   -DPACKAGE_LOCALE_DIR=\\\"\${datadir}/locale\\\" \
   -DPACKAGE_PIXMAPS_DIR=\\\"\${datadir}/pixmaps/\${PACKAGE}\\\"\
-  -DPACKAGE_LIB_DIR=\\\"\${prefix}/lib/\${PACKAGE}\\\" \
+  -DPACKAGE_LIB_DIR=\\\"\${libdir}/\${PACKAGE}\\\" \
   -DPACKAGE_ZSFB_DIR=\\\"\${DESTDIR}\${sbindir}\\\" \
 "
 
@@ -636,7 +636,7 @@
 PACKAGE_PIXMAPS_DIR="\${prefix}/\${DATADIRNAME}/pixmaps/\${PACKAGE}"
 AC_SUBST(PACKAGE_PIXMAPS_DIR)
 
-PACKAGE_LIB_DIR="\${prefix}/lib/\${PACKAGE}"
+PACKAGE_LIB_DIR="\${libdir}/\${PACKAGE}"
 AC_SUBST(PACKAGE_LIB_DIR)
 
 PLUGIN_DEFAULT_DIR="\${PACKAGE_LIB_DIR}/plugins"
