diff -Nur zapping-0.5.9.orig/Makefile.am zapping-0.5.9/Makefile.am
--- zapping-0.5.9.orig/Makefile.am	Tue Feb  6 02:10:23 2001
+++ zapping-0.5.9/Makefile.am	Sat Aug 11 00:30:31 2001
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-Multimediadir = $(datadir)/gnome/apps/Multimedia
+Multimediadir = $(datadir)/applnk/Multimedia
 Multimedia_DATA = zapping.desktop
 
 EXTRA_DIST = $(Multimedia_DATA) \
diff -Nur zapping-0.5.9.orig/zapping_setup_fb/Makefile.am zapping-0.5.9/zapping_setup_fb/Makefile.am
--- zapping-0.5.9.orig/zapping_setup_fb/Makefile.am	Tue Feb  6 02:20:50 2001
+++ zapping-0.5.9/zapping_setup_fb/Makefile.am	Sat Aug 11 00:30:09 2001
@@ -8,7 +8,7 @@
 install-binPROGRAMS: $(bin_PROGRAMS)
 if SUID_ZSFB
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
-	$(INSTALL) -m4755 -o root -g root zapping_setup_fb $(DESTDIR)$(bindir)
+	$(INSTALL) zapping_setup_fb $(DESTDIR)$(bindir)
 else
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
 	$(mkinstalldirs) $(DESTDIR)$(sbindir)
