diff -Nur plugins.orig/lirc/README plugins/lirc/README
--- plugins.orig/lirc/README	Fri Mar  2 10:59:17 2001
+++ plugins/lirc/README	Sat Aug 11 00:33:45 2001
@@ -2,7 +2,7 @@
 ----------------------------------------------
 
 In order to use the LIRC plugin the LIRC daemon must be running and the
-user must have read/write-access to /dev/lircd.
+user must have read/write-access to /tmp/.lircd.
 
 The buttons you can specify are listed in your lircd.conf file. The button
 names are case sensitive.
diff -Nur plugins.orig/lirc/lirc.c plugins/lirc/lirc.c
--- plugins.orig/lirc/lirc.c	Fri Mar  9 22:33:40 2001
+++ plugins/lirc/lirc.c	Sat Aug 11 00:33:59 2001
@@ -444,7 +444,7 @@
 int init_socket()
 {
   addr.sun_family=AF_UNIX;
-  strcpy(addr.sun_path, "/dev/lircd");
+  strcpy(addr.sun_path, "/tmp/.lircd");
   fd=socket(AF_UNIX, SOCK_STREAM, 0);
   if(fd == -1)  {
     perror("socket");
