diff -urN zapping.orig/plugins/lirc/README zapping/plugins/lirc/README
--- zapping.orig/plugins/lirc/README	Fri Oct 11 19:30:30 2002
+++ zapping/plugins/lirc/README	Sat Oct 12 23:16:11 2002
@@ -2,7 +2,7 @@
 ----------------------------------------------
 
 In order to use the LIRC plugin the LIRC daemon must be running and the
-user must have read/write-access to /dev/lircd.
+user must have read/write-access to /tmp/.lircd.
 
 The buttons you can specify are listed in your lircd.conf file. The button
 names are case sensitive.
diff -urN zapping.orig/plugins/lirc/lirc.c zapping/plugins/lirc/lirc.c
--- zapping.orig/plugins/lirc/lirc.c	Fri Oct 11 19:30:30 2002
+++ zapping/plugins/lirc/lirc.c	Sat Oct 12 23:16:38 2002
@@ -64,7 +64,7 @@
 int init_socket(void)
 {
   addr.sun_family=AF_UNIX;
-  strcpy(addr.sun_path, "/dev/lircd");
+  strcpy(addr.sun_path, "/tmp/.lircd");
   fd=socket(AF_UNIX, SOCK_STREAM, 0);
   if(fd == -1)  {
     perror("socket");
