--- zapping-0.7cvs6/configure.in.orig	2003-12-17 07:34:24.000000000 +0100
+++ zapping-0.7cvs6/configure.in	2004-04-10 13:51:52.385406998 +0200
@@ -604,7 +604,7 @@
   -DPACKAGE_DATA_DIR=\\\"\${prefix}/\${DATADIRNAME}\\\" \
   -DPACKAGE_LOCALE_DIR=\\\"\${prefix}/\${DATADIRNAME}/locale\\\" \
   -DPACKAGE_PIXMAPS_DIR=\\\"\${prefix}/\${DATADIRNAME}/pixmaps/\${PACKAGE}\\\"\
-  -DPACKAGE_LIB_DIR=\\\"\${prefix}/lib/\${PACKAGE}\\\" \
+  -DPACKAGE_LIB_DIR=\\\"\${libdir}/\${PACKAGE}\\\" \
   -DPACKAGE_ZSFB_DIR=\\\"\${DESTDIR}\${sbindir}\\\" \
 "
 
@@ -614,7 +614,7 @@
 PACKAGE_PIXMAPS_DIR="\${prefix}/\${DATADIRNAME}/pixmaps/\${PACKAGE}"
 AC_SUBST(PACKAGE_PIXMAPS_DIR)
 
-PACKAGE_LIB_DIR="\${prefix}/lib/\${PACKAGE}"
+PACKAGE_LIB_DIR="\${libdir}/\${PACKAGE}"
 AC_SUBST(PACKAGE_LIB_DIR)
 
 PLUGIN_DEFAULT_DIR="\${PACKAGE_LIB_DIR}/plugins"
