diff -urN zapping-0.6.5.org/Makefile.am zapping-0.6.5/Makefile.am
--- zapping-0.6.5.org/Makefile.am	Fri Oct 11 19:11:48 2002
+++ zapping-0.6.5/Makefile.am	Fri Oct 11 19:21:08 2002
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-Multimediadir = $(datadir)/gnome/apps/Multimedia
+Multimediadir = $(datadir)/applnk/Multimedia
 Multimedia_DATA = zapping.desktop
 
 EXTRA_DIST = config.rpath  config.rpath  $(Multimedia_DATA) \
diff -urN zapping-0.6.5.org/zapping_setup_fb/Makefile.am zapping-0.6.5/zapping_setup_fb/Makefile.am
--- zapping-0.6.5.org/zapping_setup_fb/Makefile.am	Fri Oct 11 19:11:48 2002
+++ zapping-0.6.5/zapping_setup_fb/Makefile.am	Fri Oct 11 19:32:33 2002
@@ -10,8 +10,9 @@
 install-binPROGRAMS: $(bin_PROGRAMS) zsfb.capps
 if SUID_ZSFB
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
-	$(INSTALL) -m4755 -o root -g root zapping_setup_fb $(DESTDIR)$(sbindir)
-	ln -sf $(DESTDIR)$(sbindir)/zapping_setup_fb \
+	$(mkinstalldirs) $(DESTDIR)$(sbindir)
+	$(INSTALL) zapping_setup_fb $(DESTDIR)$(sbindir)
+	ln -sf $(sbindir)/zapping_setup_fb \
 		$(DESTDIR)$(bindir)/zapping_setup_fb
 else
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
