diff -urN zapping-0.7.0.orig/zapping_setup_fb/Makefile.am zapping-0.7.0/zapping_setup_fb/Makefile.am
--- zapping-0.7.0.orig/zapping_setup_fb/Makefile.am	Sun Feb 16 21:10:00 2003
+++ zapping-0.7.0/zapping_setup_fb/Makefile.am	Sun Feb 16 21:13:42 2003
@@ -12,8 +12,7 @@
 install-binPROGRAMS: $(bin_PROGRAMS) zsfb.capps
 if SUID_ZSFB
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
-	$(INSTALL) -m4755 -o $(ZSFB_OWNER) -g $(ZSFB_GROUP) \
-	  zapping_setup_fb $(DESTDIR)$(sbindir)
+	$(INSTALL) zapping_setup_fb $(DESTDIR)$(sbindir)
 	ln -sf $(DESTDIR)$(sbindir)/zapping_setup_fb \
 	  $(DESTDIR)$(bindir)/zapping_setup_fb
 else
