diff -urN zapping-0.7.0.orig/configure.in zapping-0.7.0/configure.in
--- zapping-0.7.0.orig/configure.in	Sun Feb 16 21:10:00 2003
+++ zapping-0.7.0/configure.in	Sun Feb 16 21:19:03 2003
@@ -434,7 +434,7 @@
 DEFAULT_CFLAGS="-g -Wall -Wunused -Wmissing-prototypes -Wmissing-declarations -D_GNU_SOURCE -DGTK_DISABLE_DEPRECATED -DGNOME_DISABLE_DEPRECATED -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED $X_CFLAGS"
 AC_SUBST(DEFAULT_CFLAGS)
 
-dnl AM_PROG_AS
+AM_PROG_AS
 
 dnl AS=$CC
 dnl ASFLAGS=$DEFAULT_CFLAGS
